extra-network/clash/autobuild/overrides/usr/lib/systemd/system/clash@.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=A rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols for %i.
```

Comment:
Two "for"'s in a sentence is confusing. Also drop "A" at the beginning and period point at the end (this is purely customary with systemd units), as you will be seeing...

```
Starting rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols for %i ...
```

Or


```
Stopping rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols for %i ...
```